### PR TITLE
added java.io.Serializable to the response model #366

### PR DESCRIPTION
**Pull request by lfochi**
This is a pull request for issue #366

### CODE REVIEW

**Comment by lfochi** [comment]
I signed it!

File: src/main/java/com/google/maps/model/AddressComponent.java [hunk 1]
```diff
@@ -23,7 +24,10 @@
  * href="https://developers.google.com/maps/documentation/geocoding/intro">Google Maps Geocoding API
  * Developer's Guide</a> for more detail.
  */
-public class AddressComponent {
+public class AddressComponent implements Serializable {
+  /** The default serialVersionUID */
+  private static final long serialVersionUID = 1L;
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/model/AddressComponent.java]: Hey @markmcd and @samthor, is this a legit way of introducing `serialVersionUID` for a class that is just now implementing `Serializable`? I believe this shouldn't break anyone, but I'd like confirmation... =)

Comment (samthor) [reply src/main/java/com/google/maps/model/AddressComponent.java]: I think this is fine. It only mattered if `AddressComponent` was previously `Serializable`, which was not.

FWIW, my limited reading suggests that the reason most `serialVersionUID` fields are random super high numbers is that IDEs will create them using the previous default Java hashing implementation. But again, as the class was never serializable, anything is fine here.

File: src/main/java/com/google/maps/model/AddressComponent.java [hunk 2]
```diff
@@ -23,7 +24,10 @@
  * href="https://developers.google.com/maps/documentation/geocoding/intro">Google Maps Geocoding API
  * Developer's Guide</a> for more detail.
  */
-public class AddressComponent {
+public class AddressComponent implements Serializable {
+  /** The default serialVersionUID */
```
Comment (samthor) [inline src/main/java/com/google/maps/model/AddressComponent.java]: I think this comment is probably redundant (on all classes).

File: src/main/java/com/google/maps/model/AddressComponent.java (hunk 1 above)
Comment (domesticmouse) [reply src/main/java/com/google/maps/model/AddressComponent.java]: Thanks Sam, I concur.

File: src/main/java/com/google/maps/model/AddressComponent.java (hunk 2 above)
Comment (lfochi) [reply src/main/java/com/google/maps/model/AddressComponent.java]: Thanks Sam, I concur but as this was the first time we introduce Serializable, I thought that this comment was neccessary. But I also agree with you that probably it's redundant in every class.
Should I delete them before you aprobed this pull request? 
I tried to make real notice that that field was introduced for the serializable interface

Comment (domesticmouse) [reply src/main/java/com/google/maps/model/AddressComponent.java]: @lfochi please remove, and then I'll merge. Thanks!

**Comment by domesticmouse** [comment]
Thanks @lfochi !

**Pull request merged**